src/BuildIntegration/Microsoft.NETCore.Native.targets

```diff
@@ -23,6 +23,8 @@ See the LICENSE file in the project root for more information.
     <!-- Workaround for lack of current host OS detection - https://github.com/Microsoft/msbuild/issues/539 -->
```

Comment:
BTW: This issue was fixed - can this be changed to use `[MSBuild]::IsOSPlatform()` now?